rpcd/playbooks/roles/rpc_maas/library/raxmon.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ConfigParser
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def _get_agent_tokens(conn, entity):
+    agent_tokens = []
+    for a in conn.list_agent_tokens():
+        if a.label == entity:
+            agent_tokens.append(a)
+    return agent_tokens
+
+
+def _get_conn(get_driver, Provider, raxmon_cfg):
+    cfg = ConfigParser.RawConfigParser()
+    cfg.read(raxmon_cfg)
```

Comment:
There may be some ordering issues using /root/.raxrc as the rpc_maas role creates that file, so any use of the library before that file is created will fail. The current usage is fine because .raxrc is dropped by raxmon_cli_config which comes before the usage in agent_install_remote.

I like the simplicity of just passing the config file, but think its location should be optionally configurable. 

Do we actually use both forms of authentication? If not could we remove the one we don't use, then it may make morse sense just to pass in the relevant credentials to module.